lib/form_data.js

```diff
@@ -45,6 +45,8 @@ FormData.prototype.append = function(field, value, options) {
   if (typeof options == 'string') {
     options = {filename: options};
   }
+  if( !options.hasOwnProperty('basename') )
+    options.basename = true;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'options'.](https://www.codacy.com/app/form-data/form-data/file/2240808266/issues/source?bid=2805153&fileBranchId=3179066#l49)